pkg/kubelet/volume/gce_pd/mount_util.go

```diff
@@ -35,19 +39,36 @@ func getMountRefCount(mounter mount.Interface, mountPath string) (string, int, e
 
 	// Find the device name.
 	deviceName := ""
-	for i := range mps {
-		if mps[i].Path == mountPath {
-			deviceName = mps[i].Device
+	for _, mp := range mps {
+		if mp.Path == mountPath {
+			deviceName = mp.Device
 			break
 		}
 	}
 
 	// Find the number of references to the device.
 	refCount := 0
-	for i := range mps {
-		if mps[i].Device == deviceName {
+	for _, mp := range mps {
+		if mp.Device == deviceName {
 			refCount++
 		}
 	}
 	return deviceName, refCount, nil
 }
+
+// getPDName finds the name of the GCE PD represented by deviceName.
+func getPDName(globalMountPath string, mounter mount.Interface, deviceName string) (string, error) {
+	mps, err := mounter.List()
+	if err != nil {
+		return "", err
+	}
+	// Find the global mount point for the specified device. Should be something
+	// like /var/lib/kubelet/plugins/kubernetes.io/gce-pd/mounts/my-gce-pd.
+	// We want to return the pd name (my-gce-pd).
+	for _, mp := range mps {
```

Comment:
same comment as elsewhere - prefer to iterate lists of structs by index because of copies.